Objective-C/CBLDatabase.mm

```diff
@@ -319,7 +319,7 @@ - (BOOL) purgeDocumentWithID: (NSString*)documentID error: (NSError**)error {
 
 #pragma mark - BATCH OPERATION
 
```

Comment:
This is an addition to the public API?  I would like to do the same thing.  Pasin?

Reply:
this is only internal, not public.

Reply:
Ah.  We should put it on the agenda for 3.0 changes.  I've wanted to do the same thing.

Reply:
I think Java has already handled it correctly. Only Swift has an issue.

Reply:
Yes.  The point is that Java does not allow the lambda to throw.  It should. I've opened a ticket for that.

Reply:
Swift & ObjC

Reply:
I see. That is right.